contracts/subdomainregistrar/README.md

```diff
@@ -0,0 +1,72 @@
+# Subdomain Registrars
+
+** Note: This subdomain registrar contract only works with wrapped ENS name **
+
+The Subdomain registrar is an example contract, that can be deployed to allow registration of subnames under one or more names. The functionality is intentionally simple to allow more complicated to be custom logic to be layered on top of the basic subdomain registration functionality.
+
+The registrars layer on top of the BaseSubdomainRegistrar contract to allow additional functionality such as renting or forever subnames. They also restrict it to names that have called setupDomain with active set to true to not accidentally allow any names to have subnames registered underneath it. This means that `setApprovalForAll` will not automatically allow the subdomain contract to On top of that, each registrar takes a pricer that can control the pricing of a specific name in more granular detail.
```

Comment:
What is the usecase when a user call `setupDomain` with `active` set to `false`? Also it's missing the test case of calling the `setupDomain` with `active` set to `false`